src/views/instances/new-dhis2/helpers.tsx

```diff
@@ -0,0 +1,49 @@
+import { AnyObject } from 'react-final-form'
+import { DeployInstanceRequest } from '../../../types'
+import { OPTIONAL_FIELDS } from './constants'
+
+export const isRequired = (name: string) => !OPTIONAL_FIELDS.has(name)
+
+export const toTitleCase = (str: string): string =>
+    str
+        .toLowerCase()
+        .replace(/^[-_]*(.)/, (_, c) => c.toUpperCase())
+        .replace(/[-_]+(.)/g, (_, c) => ' ' + c.toUpperCase())
+
+export const mapStringToValueLabel = (str: string): { value: string; label: string } => ({ value: str, label: str })
+
+export const convertValuesToPayload = (values: AnyObject) =>
```

Comment:
This helper is only used in one place. Would it make sense to move it closer to where it's used?

Reply:
Moved